CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ for Rust libraries in [RFC #1105](https://github.com/rust-lang/rfcs/blob/master/
   those wishing to avoid syntax extensions from `diesel_codegen`. See
   http://docs.diesel.rs/diesel/macro.Queryable!.html for details.
 
+* The `Identifiable!` macro can now be used instead of `#[derive(Identifiable)]` for
+  those wishing to avoid syntax extensions from `diesel_codegen`. See
+  http://docs.diesel.rs/diesel/macro.Identifiable!.html for details.
```

Comment:
I had to look up whether you could use an exclamation point as part of an URL. Turns out you can ([RFC 1738, page 3](http://www.ietf.org/rfc/rfc1738.txt)) and I don't need to file a bug with rustdoc :)

Oh, and I would probably turn this into the nicer

> See [the documentation](http://docs.diesel.rs/diesel/macro.Identifiable!.html) for details.

But that's totally not important.